src/System.Console/src/System/ConsolePal.Unix.cs

```diff
@@ -73,6 +73,36 @@ public static void ResetColor()
             }
         }
 
+        public static int WindowWidth
+        {
+            get
+            {
+                int cols = Interop.Sys.GetWindowWidth();
+                return (cols != -1) ? cols : TerminalBasicInfo.Instance.ColumnFormat;
+            }
+            set
+            {
+                throw new PlatformNotSupportedException();
+            }
+        }
+
+        public static bool CursorVisible
+        {
+            get
+            {
+                throw new PlatformNotSupportedException();
+            }
+            set
+            {
+                if (!ConsoleOutIsTerminal) return;
+                string formatString = value ? TerminalBasicInfo.Instance.CursorVisibleFormat : TerminalBasicInfo.Instance.CursorInvisibleFormat;
```

Comment:
Nit: would you mind formatting this slightly differently, as:
```C#
string formatString = value ?
    TerminalBasicInfo.Instance.CursorVisibleFormat :
    TerminalBasicInfo.Instance.CursorInvisibleFormat;
```
I think that just makes it a bit easier to understand.